enterprise/coderd/license/license_test.go

```diff
@@ -102,6 +102,34 @@ func TestEntitlements(t *testing.T) {
 			fmt.Sprintf("%s is enabled but your license for this feature is expired.", codersdk.FeatureAuditLog.Humanize()),
 		)
 	})
+	t.Run("Expiration warning", func(t *testing.T) {
```

Comment:
nit: maybe add one more test for the `1 day` case.

Reply:
Fixed!